src/jvm/test/org/jetbrains/kotlinx/lincheck_test/gpmc/PrintingTest.kt

```diff
@@ -21,26 +20,30 @@ import kotlin.concurrent.thread
 class PrintingTest {
 
     /**
-     * This test would pass if `println`'s were not used.
+     * This aims to test the effects of printing to `stdout` and `stderr`.
+     * The test used to hang because of prints, see https://github.com/JetBrains/lincheck/issues/409.
+     *
      * See [CyclicBarrierTest] for identical test but with no printing.
      */
-    @Ignore("Println's hang the execution")
     @Test(timeout = TIMEOUT)
     fun testPrints() {
         runConcurrentTest(1) {
             val barrier = CyclicBarrier(2) {
-                println("All threads have reached the barrier. Proceeding...")
+                System.err.println("All threads have reached the barrier. Proceeding...")
```

Comment:
It makes sense to add tests both for regular Kotlin's `println` as well as `System.our` and `System.err`.